dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/interruptedjobwindow/JobInfoPresenter.java

```diff
@@ -258,6 +259,10 @@ protected Object call() {
                     ds3CancelSingleJobTask.setOnFailed(SafeHandler.logHandle(cancelJobTaskFailedEvent -> {
                         LOG.info("Cancellation of interrupted job " + jobId + " failed");
                     }));
+                    ds3CancelSingleJobTask.setOnFailed(SafeHandler.logHandle(event -> {
+                        LOG.error("Cancellation of recoved job faile");
```

Comment:
Seems like a typeo in here.  The text probably should read: "Cancellation of recovered job failed"

Also the exception is not being logged.  Get that from the event.